bip-0039/german.txt

```diff
@@ -0,0 +1,2048 @@
+BIRNE
+BISTRO
+BITTE
+BITUMEN
+BIWAK
+BIZEPS
+BLASE
+BLATT
+BLAUWAL
+BLEICHE
+BLESSUR
+BLICK
+BLINKER
+BLITZ
+BLOCKADE
+BLUME
+BODEN
+BOHNE
+BOHRER
+BOILER
+BOJE
+BOLLWERK
+BOLZEN
+BOMBE
+BONSAI
+BOOT
+BORD
+BORSTE
+BOSHEIT
+BOTTICH
+BOYKOTT
+BRANDUNG
+BRATEN
+BRAUEREI
+BREMSE
+BRENNGUT
+BRETT
+BREZE
+BRIKETT
+BRILLE
+BRISANZ
+BROTLAIB
+BRUCH
+BRUNNEN
+BRUST
+BRUTZEIT
+BUBE
+BUCHSE
+BUCKEL
+BUG
+BUMERANG
+BUSFAHRT
+BUSREISE
+BUSSARD
+BUTAN
+CABRIO
+CELLO
+CELSIUS
+CENT
+CHEMIKER
+CHILI
+CHOR
+CHROM
+COCKTAIL
+COUSINE
+CURRY
+DACH
+DAHEIM
+DAMENRAD
+DAMPF
+DANK
+DARLEHEN
+DATEI
+DAUERTON
+DAUNE
+DEBATTE
+DECKEL
+DEFEKT
+DEFIZIT
+DEGEN
+DEICHSEL
+DEKADE
+DEKOR
+DELFIN
+DELIKT
+DELLE
+DEMENTI
+DENKMAL
+DEPONIE
+DESASTER
+DESIGNER
+DETEKTOR
+DEUTUNG
+DEVISE
+DEZEMBER
+DIAGNOSE
+DIALOG
+DICKICHT
+DIDAKTIK
+DIELE
+DIENST
+DIKTAT
+DILL
+DIODE
+DIOPTRIE
+DIPLOMAT
+DIREKTOR
+DIRIGENT
+DISKURS
+DISTEL
+DIVISION
+DOCHT
+DOGGE
+DOHLE
+DOLCH
+DOLDE
+DOLLAR
+DOMIZIL
+DOMPTEUR
+DORADE
+DORF
+DORSCH
+DOSIERER
+DOZENT
+DRACHE
+DRAHT
+DRALL
+DRANG
+DRECK
+DREHBUCH
+DREIECK
+DRESSUR
+DROHUNG
+DROMEDAR
+DRUCKER
+DUELL
+DUETT
+DUFT
+DUNG
+DUPLIKAT
+DURST
+DUSCHE
+DUTZEND
+DYNAMIK
+EBBE
+EBENE
+EBER
+ECHOLOT
+ECHTZEIT
+ECKBALL
+ECKE
+EDAMER
+EDELGAS
+EDITION
+EFEU
+EFFEKT
+EGERLING
+EGO
+EHEBETT
+EHEPAAR
+EHRE
+EHRGEIZ
+EIDECHSE
+EIGELB
+EIGNUNG
+EILTEMPO
+EIMER
+EINBRUCH
+EINDRUCK
+EINFUHR
+EINGRIFF
+EINHORN
+EINIGUNG
+EINKAUF
+EINLAGE
+EINNAHME
+EINRAD
+EINSATZ
+EINTOPF
+EINWAND
+EINZAHL
+EISDIELE
+EISEN
+EISFACH
+EISKANAL
+EISTANZ
+EISWEIN
+EISZEIT
+EIZELLE
+EKLAT
+EKSTASE
+ELAN
+ELCH
+ELEFANT
+ELEGANZ
+ELEKTRIK
+ELFMETER
+ELLBOGEN
+ELOQUENZ
+ELSTER
+ELTERN
+EMBLEM
+EMBRYO
+EMISSION
+EMPATHIE
+EMPFANG
+EMPORE
+EMULSION
+ENDLAGER
+ENDOSKOP
+ENDPHASE
+ENDSPURT
+ENERGIE
+ENGPASS
+ENKLAVE
+ENTGELT
+ENTROPIE
+ENTWURF
+ENZIAN
+ENZYM
+EPOCHE
+ERBAUUNG
+ERBFOLGE
+ERBGUT
+ERBPACHT
+ERBSE
+ERDAPFEL
+ERDBODEN
+ERDGAS
+ERDKUGEL
+ERDNUSS
+ERDREICH
+ERDTEIL
+ERDUNG
+ERFINDER
+ERFOLG
+ERGEBNIS
+ERHEBUNG
+ERLASS
+ERLE
+ERNST
+ERNTE
+EROBERER
+ERPEL
+ERREGER
+ERSATZ
+ERTRAG
+ERUPTION
+ERZ
+ESCHE
+ESELSOHR
+ESKAPADE
+ESKORTE
+ESSENZ
+ESSIG
+ESSTISCH
+ESTRICH
+ETAGE
+ETAPPE
+ETAT
+ETHIK
+ETIKETT
+ETUI
+EUTER
+EVIDENZ
+EWIGKEIT
+EXEMPEL
+EXIL
+EXISTENZ
+EXKURS
+EXOT
+EXPANDER
+EXPERTE
+EXPONAT
+EXZESS
+FABRIK
+FACHWERK
+FACKEL
+FADEN
+FAGOTT
+FAHNE
+FAHRRAD
+FAKIR
+FAKTOR
+FALKE
+FALLOBST
+FAMILIE
+FANFARE
+FANPOST
+FANTASIE
+FARBTON
+FASAN
+FASCHING
+FASSADE
+FAULHEIT
+FAUST
+FAZIT
+FEBRUAR
+FEDERUNG
+FEHLER
+FEIERTAG
+FEIGHEIT
+FEILE
+FEIND
+FELDWEG
+FELGE
+FELL
+FELSWAND
+FENCHEL
+FENSTER
+FERIEN
+FERKEL
+FESTUNG
+FETT
+FETUS
+FEUER
+FIASKO
+FIBEL
+FIGUR
+FILET
+FILIALE
+FILZ
+FINALE
+FINDLING
+FINK
+FINNE
+FISKUS
+FLAGGE
+FLANSCH
+FLASCHE
+FLAUTE
+FLECK
+FLEISCH
+FLIESE
+FLINTE
+FLOH
+FLOSKEL
+FLOTTE
+FLUCHT
+FLUGZEUG
+FLUNDER
+FLUR
+FLUSS
+FOHLEN
+FOKUS
+FORELLE
+FORMAT
+FORST
+FORTGANG
+FOTOGRAF
+FOUL
+FRACHTER
+FRAGMENT
+FRAKTUR
+FRAU
+FREGATTE
+FREITAG
+FREQUENZ
+FRESKE
```

Comment:
```suggestion
FRESKO
```
Masculine version seems to be used predominantly here, didn't know there's a feminine version.
![image](https://user-images.githubusercontent.com/1100176/108708843-a2735900-751a-11eb-87a1-6129f2d68e1e.png)
![image](https://user-images.githubusercontent.com/1100176/108708906-b5862900-751a-11eb-84e7-b73e9037caec.png)

Reply:
OK, let's go for "Fresko" except you'd like to replace it completely.

Reply:
@SebastianFloKa "Fresko" sounds good and is already replaced, this thread can be marked as resolved too.

Reply:
Word changed